operators/operator_sdk/java/osdk-java-tutorial.adoc

```diff
@@ -0,0 +1,69 @@
+Java-based:_content-type: ASSEMBLY
```

Comment:
Looks like a stray bit of text in here, it's causing the assembly to not render properly. Can you fix this before I go through the rest of this file please?